components/KitchenSink.tsx

```diff
@@ -430,39 +432,51 @@ export const KitchenSink: React.FC = (): JSX.Element => {
                         icon={{ family: 'brightlayer-ui', name: 'device' }}
                         iconColor="green"
                     />
-                    {/* Default Font size : 14px */}
+                    {/* Font size : 14px */}
                     <ChannelValue value="2.5:1" icon={{ name: 'settings' }} fontSize={14} style={{ marginTop: 12 }} />
-                    {/* Default Font size : 16px */}
-                    <ChannelValue value="Concord" icon={'🍇'} style={{ marginTop: 12 }} />
-                    {/* Default Font size : 22px */}
+                    {/* Font size : 16px */}
```

Comment:
Here the comment says `Font size : 16px` and in example we are passing fontSize={14}. We may need to update one of those.